lib/web/websocket/frame.js

```diff
@@ -2,26 +2,40 @@
 
 const { maxUnsigned16Bit } = require('./constants')
 
+const BUFFER_SIZE = 16386
+
 /** @type {import('crypto')} */
 let crypto
+let buffer = null
+let bufIdx = BUFFER_SIZE
+
 try {
   crypto = require('node:crypto')
 /* c8 ignore next 3 */
 } catch {
 
 }
 
+function generateMask () {
+  if (bufIdx === BUFFER_SIZE) {
+    bufIdx = 0
+    crypto.randomFillSync((buffer ??= Buffer.allocUnsafe(BUFFER_SIZE)), 0, BUFFER_SIZE)
```

Comment:
This will throw if there is no crypto.

Reply:
Why? It is not called when there is no crypto.

Reply:
@KhafraDev Which choice would you prefer?

Reply:
I'd probably just use Math.random. We can take a look at disabling masking at another time.